vuepress/docs/next/tutorials/ecr/github-actions-workflow.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebarDepth: 2
+---
+
+# Add a GitHub Actions CI Workflow
+
+## Overview
+This tutorial shows how to use the JHipster `ci-cd` sub-generator to quickly create a basic GitHub Actions Workflow for the microservices and micro frontends (MFEs) generated by the Entando Component Generator.
+
+## Prerequisites
+* An existing project with a GitHub repository. See the [Entando Component Generator](../backend-developers/generate-microservices-and-micro-frontends.md#overview) to create a new one.
+* [GitHub Actions](https://github.com/features/actions). These are enabled by default without additional configuration. Please note public repositories currently qualify for unlimited usage but private repositories can encounter usage restrictions.
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) to verify you have the command line prerequisites in place for this tutorial (e.g. npm, git, JHipster).
+
+## Create a backend workflow
+
+1. Go to your main project folder in the shell
+2. Run the JHipster `ci-cd` subgenerator
+```bash
+ent jhipster ci-cd
+```   
+2. Select `GitHub Actions`
+3. Click ENTER to choose no `tasks/integrations` and generate the default workflow
```

Comment:
should workflow be capitalized?

Reply:
I don't think so since the folder the file is created in is lowercase. GitHub doesn't capitalize it in general - https://docs.github.com/en/actions/learn-github-actions